docs/www/quick-start-kubernetes.md

```diff
@@ -186,304 +186,9 @@ Let's try setting up a Redpanda topic to handle a stream of events from a chat a
-
-```
-$ kubectl exec external-connectivity -- cat /etc/redpanda/redpanda.yaml 
-  advertised_kafka_api:
-    - address: external-connectivity-0.external-connectivity.default.svc.cluster.local.
-      name: kafka
-      port: 9092
-    - address: <external-node-ip>
-      name: kafka-external
-      port: 31848
-```
-
-### Container configuration
-
-The Redpanda operator ensures that the Redpanda container exposes the requested ports, as described in the CR. The Redpanda containers are configured to expose two ports per API - one internal and one external. The operator does not create an external port for RPC.
-
-By running `kubectl describe pod external-connectivity-0` we can see the ports and the mapping to the node ports created through the nodeport service:
-
-```
-$ kubectl describe pod external-connectivity-0 
-...
-Containers:
-  redpanda:
-    ..
-    Ports:         33145/TCP, 9644/TCP, 9092/TCP, 8082/TCP, 9093/TCP, 9645/TCP, 8083/TCP
-    Host Ports:    0/TCP, 0/TCP, 0/TCP, 0/TCP, 31848/TCP, 31490/TCP, 30638/TCP
-```
-
-### Port Summary - Configuration based on example specification
-
-|   | Internal listener ports  | External listener ports (port+1:nodeport)  |
-|---|---|---|
-| Admin API  | 9644  | 9645:31490  |
-| Kafka API  | 9092  | 9093:31848  |
-| Pandaproxy API  | 8082  | 8083:30638  |
-
-
-### Using names instead of external IPs
-
-The CRD includes a field “subdomain” that allows to specify the advertised address of external listeners. Here’s an example for the Kafka API:
-```
-  configuration:
-    kafkaApi:
-    - port: 9092
-    - external:
-        enabled: true
-        subdomain: "test.subdomain.com"
-```
-
-The generated `redpanda.yaml` configuration uses the subdomain field to generate the advertised addresses for the external listeners following this format: \<broker_id\>.\<subdomain\>:\<node_port\>. Note that the DNS configuration is *not* handled by the Redpanda operator.
-
-The Redpanda configuration will reflect this in the advertised addresses: 
-
-```
-$ kubectl exec external-connectivity -- cat /etc/redpanda/redpanda.yaml 
-...
-redpanda:
-  advertised_kafka_api:
-    - address: external-connectivity-0.external-connectivity.default.svc.cluster.local.
-      name: kafka
-      port: 9092
-    - address: 0.test.subdomain.com
-      name: kafka-external
-      port: 31631
-```
-
-Finally, the CR contains the addresses in its status:
-
-```
-$ kubectl describe cluster external-connectivity
-...
-Status:
-  Nodes:
-    External:
-      0.test.subdomain.com:31631
-      1.test.subdomain.com:31631
-      2.test.subdomain.com:31631
-```
+As you can see, the commands from the "rpk" pod created a 5-partition topic in for the chat rooms.
 
 ## Next steps
 
+- Check out our in-depth explanation of [Kubernetes connectivity](/docs/kubernetes-connectivity).
```

Comment:
It isn't kubernetes connectivity tho. that's It is `how to connect outside clients to redpanda` or smth like that. kubernetes connectivity is much larger topic. This is just for the Redpanda Operator (product)

Reply:
How's: Check out our in-depth explanation of how to [connect external clients](/docs/kubernetes-connectivity) to a Redpanda Kubernetes deployment.